spec/arena_spec.rb

```diff
@@ -6,9 +6,33 @@
 require './lib/arena'
 require './lib/gladiator'
 
+module Helpers
```

Comment:
Was wondering how I could create reusable methods and found [this helpful link](http://testdrivenwebsites.com/2011/08/17/different-ways-of-code-reuse-in-rspec/).